src/wagtail_content_import/pickers/microsoft/wagtail_hooks.py

```diff
@@ -22,7 +26,7 @@ def create_from_microsoft_doc(request, parent_page, page_class):
     if "microsoft-doc" in request.POST and not is_importing(request):
         set_importing(request)
         document_url = request.POST["microsoft-doc"]
-        response = requests.get(document_url)
+        response = requests.get(document_url, timeout=25)
```

Comment:
adds a timeout for requests. I've set it to 25 on the premise that Heroku has a 30s timeout.

Reply:
Nitpick: Noticed we use this a few places - a setting is probably overkill, but could we make the timeout a constant so we can easily tweak in one place?